src/Data/Array.js

```diff
@@ -7,7 +7,7 @@
 exports.range = function (start) {
   return function (end) {
     var step = start > end ? -1 : 1;
-    var result = [];
+    var result = new Array(step * (end - start) + 1);
```

Comment:
This seems wrong. Isn't this too large when `step > 1`?

Reply:
What do you mean? It can only be -1 or 1...

Reply:
Oops, sorry, I misread :man_facepalming: